ModerationPolicy.md

```diff
@@ -0,0 +1,356 @@
+# Moderation policy
+
+If you are not a member and wish to submit a
+moderation request, please see [Requesting Moderation][].
+
+## Table of Contents
+
+-   [Applicability][]
+-   [Terms][]
+-   [Grounds for Moderation][]
+-   [Requesting Moderation][]
+-   [Consideration of Intent][]
+-   [Guidelines and Requirements][]
+    -   [Collaborator Posts][]
+    -   [Non-Collaborator Posts][]
+    -   [Temporary Interaction Limits][]
+    -   [Temporary and Indefinite Blocks][]
+-   [Privacy of the expressjs/moderation Repository][]
+-   [Moderation Team][]
+-   [Escalation of Issues][]
+-   [Modifications to This Policy][]
+
+## Applicability
+
+This policy applies to all repositories under these GitHub organizations: expressjs, jshttp, and pillarjs, as well as all Working Groups.
+
+Any posts, comments, issues, or pull requests that violate the [Code of Conduct][] are subject to Moderation as defined in this policy.
```

Comment:
This policy also needs to apply to any project run comms: slack, discord, socials (the ones on behalf of the project, not personal), etc. Not just githbu.

Reply:
TIL. Thanks for that. We should then be very explicit in the wording here.

Reply:
Alright, revising that.